stylesheets/example-tooltip.scss

```diff
@@ -50,18 +50,27 @@ $shadow-color: rgba(31, 28, 28, 0.3);
     .btn {
       font-size: 13px;
       font-weight: 400;
-      color: $white;
-      background-color: #78A300;
+      color: #78A300;
+      background-color: transparent;
+      border: 1px solid #78A300;
       border-radius: 3px;
       height: 36px;
       padding: 0 20px;
+
+    }
+
+    .btn-next {
+      color: $white;
+      background-color: #78A300;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4451516/22355870/745480a4-e481-11e6-9213-e2fa2452d6ae.png)
Here is a screenshots for now, I will up update the live demo as well if this PR would get merged.